### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Kelvin-Lee**
## Test plan
What I have done: Updated documentation to follow some language/wording provided [here](https://github.com/sourcegraph/customer/issues/694) regarding current limitations of the [Stream API](https://docs.sourcegraph.com/api/stream_api).

To test: Ummm I put it through a spell checker. Small complaints about `parallelization` but Wikipedia seems to be happy that it's a word/concept.

### CODE REVIEW

File: doc/api/stream_api/index.md [hunk 1]
```diff
@@ -115,3 +115,7 @@ If you don't want to write your own client, you can also use Sourcegraph's [src-
 ```bash
 src search -stream "secret count:all"
 ```
+
+### Q: Are there plans for parallelization/convenience-wrappers/multiplexing/etc.?
+
+At time of writing, there are no such plans for parallelization/convenience-wrappers/multiplexing/etc. If a user wants these characteristics, we would recommend that user to write their own scripts that handles firing multiple requests, accepting the return values, and formatting/parsing them as needed.
```
Comment (rvantonder) [inline doc/api/stream_api/index.md:121]:
```suggestion
### Q: Are there plans for supporting a streaming client or interface with more functionality (e.g., parallelizing multiple streaming requests or aggregating results from multiple streams)?

There are currently no plans to support additional client-side functionality to interact with a streaming endpoint. We recommend users write their own scripts or client wrappers that handle, e.g., firing multiple requests, accepting and aggregating the return values, and additional result formatting or processing.
```

I think we want to make it clear that "parallelization" refers to multiple client-side requests (we do parallelize the actual search internally).

**Review by rvantonder — COMMENTED** [review]
@Kelvin-Lee shall we update and merge this? It's still hanging out in my PR review queue :)

**Comment by Kelvin-Lee** [comment]
Oh gosh sorry @rvantonder . Merged your commit, will await the build checks for a merge!

**Review by rvantonder — APPROVED** [review]
go ahead with merge :-) thanks

**Pull request merged**